.github/workflows/checks.yaml

```diff
@@ -0,0 +1,95 @@
+name: Checks
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+      - test-ci**
+  workflow_dispatch: # Enable workflow to be run manually
+
+# Disable all access to the GitHub API by default
+# This default can be overridden for individual workflows and jobs
+# Documentation: https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#permissions
+permissions: {}
+
+# Cancel the currently running CI job if you push a change while CI is running
+# Documentation: https://docs.github.com/en/actions/using-jobs/using-concurrency
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  NODEJS_VERSION: 22.14.0
+
+jobs:
+  checks:
+    runs-on: ubuntu-22.04 # Pinned version of `ubuntu-latest`
+    name: Run checks
+    env:
+      PYTHON_VERSION: 3.11.11
```

Comment:
Why is `PYTHON_VERSION` 3.11.11? iirc the diplomacy package supports version 3.6/3.7.

Reply:
All bots in `chiron-utils` are written to use Python 3.11, so we need to ensure it works on both Python 3.11 and 3.7.

I'm also planning to upgrade the server to running with Python 3.11 in a later PR, but I wanted to add CI first. It would allow us to actually get security patches for the server I am running as `root` on my personal machine...